ops/terraform/modules/resources/bfd_server_metrics/main.tf

```diff
@@ -10,8 +10,8 @@ locals {
     coverage_all       = "*/fhir/Coverage*"
     patient_all        = "*/fhir/Patient*"
     eob_all            = "*/fhir/ExplanationOfBenefit*"
-    claim_all          = "*/fhir/Claim*"
-    claim_response_all = "*/fhir/ClaimResponse*"
+    claim_all          = "*/fhir/Claim/*"
+    claim_response_all = "*/fhir/ClaimResource/*"
```

Comment:
Just double checking, since most of the story talks about ClaimResponse throughout the description but then says ClaimResource in the example... should this be ClaimResponse, or ClaimResource? It somewhat intuitively seems like it should be ClaimResource?

Reply:
I thought about that too. Changing for now and can revert it back if other people disagree

Reply:
I mixed up myself even talking about; I meant that Claim_Response_ is correct. A quick look at our endpoints shows that ClaimResponse is the correct value:
![image](https://user-images.githubusercontent.com/1200656/227365389-1f8ed590-dcb0-44a2-a8f2-0e4afc4bac49.png)

Reply:
Fixed

Reply:
FYI: https://hl7.org/fhir/claimresponse.html